index.html

```diff
@@ -4555,6 +4586,28 @@ <h2>
                   </li>
                 </ol>
               </li>
+              <li data-link-for="PaymentDetailsUpdate">If the
+              <a>paymentMethodErrors</a> member is present and
+              <var>identifier</var> is not undefined:
+                <ol>
+                  <li>If required by the specification that defines the
+                  <var>identifer</var>, then <a data-cite=
+                  "!WEBIDL#dfn-convert-ecmascript-to-idl-value">convert</a> <a>
+                    paymentMethodErrors</a> to an IDL value. Otherwise,
+                    <a data-cite=
```

Comment:
It's already an object, so no need to convert.

Reply:
Ok yeah. I thought it was still a JS object and it was being converted into an IDL object. We might need to check other parts of the spec, as we do the same thing in other places.